src/EncompassRest/Loans/RateLocks/RequestIndicators.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.RateLocks
+{
+    /// <summary>
+    /// RequestIndicators
+    /// </summary>
+    public enum RequestIndicators
```

Comment:
Could you rename to `RequestIndicator`. Plural enum names are usually reserved for Flag enums.

Reply:
Done.